library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -368,7 +370,8 @@ class Client() {
                     accountAddress = accountAddress,
                     inboxId = inboxId,
                     nonce = 0.toULong(),
-                    legacySignedPrivateKeyProto = privateKeyBundleV1.toV2().identityKey.toByteArray()
+                    legacySignedPrivateKeyProto = privateKeyBundleV1.toV2().identityKey.toByteArray(),
+                    historySyncUrl = options.historySyncUrl
```

Comment:
@jhaaaa just putting the message history stuff on your radar for documentation. Probably should wait till @tuddman gets back from vacation but we could start spiking it out in the mean time.